addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -316,10 +316,35 @@
             </div>
             <div t-if="attendees" class="row mb-3">
                 <div class="col-12 mb-2">
-                    <a class="btn btn-primary" title="Download All Tickets" target="_blank"
+                    <h5>Get your Tickets</h5>
+                </div>
+                <div class="col-12 mb-2 d-flex flex-column align-items-center flex-md-row align-items-md-baseline ">
+                    <a class="btn btn-primary" title="Download All Tickets" target="_blank" style="min-width: 9rem;"
                         t-attf-href="/event/{{ event.id }}/my_tickets?registration_ids={{ attendees.ids }}&amp;tickets_hash={{ event._get_tickets_access_hash(attendees.ids) }}">
-                        Download Tickets <i class="ms-1 fa fa-download"/>
+                        Download <i class="ms-1 fa fa-download"/>
                     </a>
+                    <span class="mx-3">or</span>
+                    <div class="o_send_by_email_widget has-validation ms-md-2 mt-2 mt-md-0" style="max-width: 20rem;">
+                        <div class="input-group">
+                            <input id="o_send_by_email_input" type="text"
+                                placeholder="john@doe.com" class="form-control"/>
+                            <button class="o_wevent_js_send_email btn btn-primary"
+                                type="button"
+                                t-att-data-event-id="event.id"
+                                t-att-data-attendees-ids="attendees.ids"
+                                t-att-data-tickets-hash="event._get_tickets_access_hash(attendees.ids)">
+                                Send by Email
+                            </button>
+                        </div>
+                        <div class="invalid-feedback">
+                            Please enter valid email(s)
+                        </div>
+                    </div>
+                    <div class="o_send_by_email_info alert alert-info ms-md-2 mt-2 mt-md-0 mb-0 px-3 py-1 d-inline-flex align-items-center justify-content-center d-none" role="alert">
```

Comment:
are all of theses necessary? :p

Reply:
As I use 'alert' & 'alert-info', I get a lot of stuff that I don't want here so I have to override it...
Not sure if we can simplify it :/

Reply:
input-group has-validation